src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $calendar-gap;
+  align-items: center;
+  padding: $calendar-padding;
+  margin:0 auto;
+  padding-top: 100px;
```

Comment:
The calendar should be centered both horizontally and vertically. Get rid of all margins and paddings here and simply center it by adding flex to body (don't forget about justify, align and height)